be/src/util/hdfs_util.cpp

```diff
@@ -6,15 +6,15 @@
 #include <hdfs/hdfs.h>
 
 #include <string>
-using std::string;
 
+#include "gutil/strings/substitute.h"
 #include "util/error_util.h"
 
 namespace starrocks {
 
-const char* FILESYS_PREFIX_HDFS = "hdfs://";
-const char* FILESYS_PREFIX_S3 = "s3a://";
-const char* FILESYS_PREFIX_OSS = "oss://";
+static const char* FILESYS_PREFIX_HDFS = "hdfs://";
```

Comment:
should start with `k` 
according to https://starrocks.feishu.cn/docs/doccnH0eI3LRYUa40tsEZjviI3c#JJSjdN